### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: lib/ZcapClient.js [hunk 1]
```diff
@@ -113,7 +113,10 @@ export class ZcapClient {
     if(url) {
       // generate the root capability
```
Comment (dlongley) [inline lib/ZcapClient.js:114]:
```suggestion
```

File: lib/ZcapClient.js [hunk 2]
```diff
@@ -123,7 +126,10 @@ export class ZcapClient {
     } else {
       delegatedCapability = {
         // use a provided capability
```
Comment (dlongley) [inline lib/ZcapClient.js:128]:
```suggestion
```

File: lib/ZcapClient.js [hunk 3]
```diff
@@ -123,7 +126,10 @@ export class ZcapClient {
...
-        '@context': SECURITY_CONTEXT_V2_URL,
+        '@context': [
+          SECURITY_CONTEXT_V2_URL,
+          'https://w3id.org/security/suites/ed25519-2020/v1',
+        ],
         id: await generateZcapUri(),
         parentCapability: capability.id,
```
Comment (dlongley) [inline lib/ZcapClient.js:134]:
```suggestion
        // use a provided capability as parent
        parentCapability: capability.id,
```

File: lib/ZcapClient.js [hunk 4]
```diff
@@ -113,7 +113,10 @@ export class ZcapClient {
     if(url) {
       // generate the root capability
       delegatedCapability = {
-        '@context': SECURITY_CONTEXT_V2_URL,
+        '@context': [
+          SECURITY_CONTEXT_V2_URL,
+          'https://w3id.org/security/suites/ed25519-2020/v1',
+        ],
         id: await generateZcapUri(),
         parentCapability: await generateZcapUri({url}),
```
Comment (dlongley) [inline lib/ZcapClient.js:121]:
```suggestion
        // use generated root capability as parent
        parentCapability: await generateZcapUri({url}),
```

File: lib/ZcapClient.js [hunk 5]
```diff
@@ -138,14 +144,24 @@ export class ZcapClient {
...
+        suite: new Ed25519Signature2020({
           signer: this.delegationSigner,
           verificationMethod: this.delegationSigner.id
         }),
         purpose: new CapabilityDelegation({
           capability,
           capabilityChain: (url) ? [await generateZcapUri({url})] : undefined
-        })
+        }),
+        // FIXME: WHAT'S THE SOLUTION FOR DOCUMENTLOADER?
```
Comment (dlongley) [inline lib/ZcapClient.js:155]: Pass a doc loader that will properly return the ed25519-2020 context.

Comment (mattcollier) [reply lib/ZcapClient.js:155]: @dlongley @dmitrizagidulin I would really like to see the ergonomics of all this improved.  Can we imagine a situation where the SignatureSuites can provide a document loader for their context and this gets automagically added to a documentLoader that is internal to jsonld-signatures.  

I also feel that having to deal with putting a context on my document specific to the signature suite is work that could be done elsewhere.  On thought that occurred to me was... Is it not possible for the `proof` to have its own context?  And therefore consumers of the `sign` API don't have to correlate the context on their documents with the signature suites.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: lib/ZcapClient.js [hunk 6]
```diff
@@ -138,14 +145,15 @@ export class ZcapClient {
 
     const signedDelegatedCapability = await jsigs.sign(
       delegatedCapability, {
-        suite: new Ed25519Signature2018({
+        suite: new Ed25519Signature2020({
           signer: this.delegationSigner,
           verificationMethod: this.delegationSigner.id
```
Comment (dlongley) [inline lib/ZcapClient.js:150]: This doesn't exist anymore: https://github.com/digitalbazaar/ed25519-signature-2020/blob/main/lib/Ed25519Signature2020.js#L45

And it looks like we can just drop it given that `id` is set on the signer: https://github.com/digitalbazaar/jsonld-signatures/blob/8a4d36ed9d4167dde5ecfe30051361115806284c/lib/suites/LinkedDataSignature.js#L344-L345

```suggestion
```

File: lib/ZcapClient.js [hunk 7]
```diff
@@ -138,14 +145,15 @@ export class ZcapClient {
...
         }),
         purpose: new CapabilityDelegation({
           capability,
           capabilityChain: (url) ? [await generateZcapUri({url})] : undefined
```
Comment (dlongley) [inline lib/ZcapClient.js:154]:
```suggestion
          capabilityChain: url ? [await generateZcapUri({url})] : undefined
```

**Review by dlongley — APPROVED** [review]
Approving w/minor comment to address.

**Comment by dlongley** [comment]
Closing. The commits here have been folded into the `fix-delegation` branch.

**Pull request closed without merging**